pgx_async/bin/jbuild

```diff
@@ -0,0 +1,8 @@
+(jbuild_version 1)
+
+(executables
+ ((names (pgx_async_example))
+  (package pgx_async)
+  (public_names (pgx_async_example))
```

Comment:
I think we probably don't want to install this with the opam package (if we ever release one..). Removing this line should fix that, and then you could add something to the `Makefile` to do:

```
jbuilder build bin/pgx_async_example.exe
```

Or something like that?